ting-bill-split.go

```diff
@@ -11,22 +11,144 @@ import (
 	"strconv"
 
 	"github.com/BurntSushi/toml"
+	"github.com/shopspring/decimal"
 )
 
 type bill struct {
-	Minutes   float64 `toml:"minutes"`
-	Messages  float64 `toml:"messages"`
-	Megabytes float64 `toml:"megabytes"`
-	Devices   float64 `toml:"devices"`
-	Extras    float64 `toml:"extras"`
-	Fees      float64 `toml:"fees"`
+	Minutes      float64  `toml:"minutes"`
+	Messages     float64  `toml:"messages"`
+	Megabytes    float64  `toml:"megabytes"`
+	Devices      float64  `toml:"devices"`
+	Extras       float64  `toml:"extras"`
+	Fees         float64  `toml:"fees"`
+	DeviceIds    []string `toml:"deviceIds"`
+	ShortStrawID string   `toml:"shortStrawId"`
+	Total        float64  `toml:"total"`
+}
+
+type billSplit struct {
+	MinSubs   map[string]decimal.Decimal
+	MsgSubs   map[string]decimal.Decimal
+	MegSubs   map[string]decimal.Decimal
+	DeltaSubs map[string]decimal.Decimal
```

Comment:
Do you feel like some renaming, or comments would be the way to go?  Or both?  I figure either way I need to put some comments above some of my structs for clarity.